src/App.css

```diff
@@ -27,6 +27,28 @@
   display: inline;
 }
 
+.App .spaceyspace {
+  height: 60px;
+}
+
+.App .exiting {
+  animation: comp-exit 2s linear;
+}
+
+.App .entering {
```

Comment:
I moved the transition back to the original classes instead of having the keyframes. I'm still confused by what you mean about the classNames (see above comment)